docs/usage.md

```diff
@@ -349,9 +909,33 @@ For example:
 
 ## Preprocessing
 
+### --aligner
+
+To control which aligner is used for mapping the reads.
+
+Available: `bwa-mem` and `bwa-mem2`
+
+Default: `bwa-mem`
+
+Example:
+
+```bash
+--aligner "bwa-mem"
+```
+
+> **WARNING** Current indices for `bwa` in AWS iGenomes are not compatible with `bwa-mem2`.
+> Use `--bwa=false` to have `Sarek` build them automatically.
```

Comment:
Since the memory requirement are currently ginormous, do you think it would make sense to add a note about them here? Like...
BWA-Mem2 is currently under active development. For information on required resources, also take a look at their [repository](https://github.com/bwa-mem2/bwa-mem2).

Reply:
That is a very good point.
Maybe say something like that:
BWA-mem2 is in active development, Sarek might not be able to require the right amount of resources for it at the moment.
We recommend to use pre-built indexes.

Reply:
Sounds great :)